_posts/2022-11-08-thudtest.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+authors:
+  - hdhalter
+title: "The thud test: Measuring OpenSearch Documentation"
+date: 2022-11-10 01:01:01 -0700
+categories:
+- feature
+---
+
+# The thud test: Measuring OpenSearch documentation 
+
+When I first started my career as a technical writer about 20 years ago, success was measured by the volume of content produced. Back then, everything was printed and bound. When you dropped a stack of paper on a desk or the ground, it would make an audible *thud*. The theory went that the louder the thud, the more content you had produced, and the happier your customers were.
+
+Fast-forward about 10 years, and it was all about simplification: Only provide the minimum amount of content that your customers need to accomplish their tasks. So we set about removing information and only documenting what was absolutely necessary.
+
+In the years that followed, technical writers continued to face tough choices. We know that we have to write for customers first. We know that writing simply is hard. We know that we need to insist on the highest standards. We know that words are expensive to translate and resource intensive to change as the user interface changes.
+
+And we know that product managers still want to use the thud test to measure documentation output. <p style="max-width: 50%; float: right;">
+![Stack of papers](../assets/media/blog-images/stack%20of%20papers.jpg)</p>
```

Comment:
looks like the file uploaded is .jpeg but this references .jpg

also, the path to your images needs to be resolved. See this example from another blog post. 

Example from another post: 
<img src="/assets/media/blog-images/2022-10-31-adding-new-distributions-in-opensearch-project/jenkins_pipeline_flowchart.png" alt="Jenkins Pipeliens Flowchart"/>{: .img-fluid }

you'll need to resolve all the image locations, and redo your path within each image line through this blog post.

Reply:
ok thanks!